src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -128,6 +128,24 @@ components:
         - en
       default: en
 
+    CertificateDelivery:
+      title: CertificateDelivery
+      description: Object describing how and where the Voter Authority Certificate should be delivered
+      type: object
+      properties:
+        deliveryOption:
+          $ref: '#/components/schemas/DeliveryOption'
+        addressee:
+          type: string
+          maxLength: 255
+          description: The addressee to be printed in addition to the delivery address
+        address:
+          $ref: '#/components/schemas/Address'
+      required:
+        - deliveryOption
+        - addressee
+        - address
+
     DeliveryOption:
```

Comment:
Is `DeliveryClass` appropriate here?  (As in First class, Second class delivery options)

Reply:
Yes, DeliveryClass is a much better name 👍 , thanks I'll do that 👍